### DISCUSSION TITLE
Documentation issues

### DISCUSSION THREAD

**Discussion by SymBevins** [boris-jenicek/ngx-awesome-popup#34] [Q&A] [answered]
Hi I recently found this library and I love it so far. The problem I am running into is when I try and go through the documentation pretty much every single page under the Api section is blank. Is this how it is? or is something going on there? 

I am having issues getting custom colors to work, and I can't read anything on the api to see what's going on. 

Any help or updates on the documentation would be appreciated. Thanks!

**Comment by SymBevins** [comment, marked as answer]
I found the docs in the repo. Seems like the links between the docs are broken. Just an FYI now. 

Thanks

**Reply by boris-jenicek** [reply]
it is fixed now on the demo site as well, thank you for reporting!
